share/sysbuild/CMakeLists.txt

```diff
@@ -16,6 +16,10 @@ set(Sysbuild_DIR ${CMAKE_CURRENT_LIST_DIR}/../sysbuild-package/cmake)
 
 project(sysbuild_toplevel LANGUAGES)
 
+if(NOT DEFINED APPLICATION_BINARY_DIR)
+  set(APPLICATION_BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR} CACHE PATH "Application Binary Directory")
```

Comment:
Seems alright, but I wonder if changing:

https://github.com/zephyrproject-rtos/zephyr/blob/c37e46c9064a7067424b27fc4a81c7a07f26c8dc/share/zephyr-package/cmake/ZephyrConfig.cmake#L51

to `CMAKE_BINARY_DIR` would've been a more general fix.

Reply:
I did consider that, but simply decided to go with same pattern as: 
https://github.com/zephyrproject-rtos/zephyr/blob/df788d0b41300473f047a1497fae57e8ab351f26/share/zephyr-package/cmake/ZephyrConfig.cmake#L50-L54

In the sysbuild case, then `CMAKE_CURRENT_BINARY_DIR` == `CMAKE_BINARY_DIR` when sysbuild is invoked as documented / through `west build`.